<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scanner</class>
 <widget class="QWidget" name="Scanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="extoption">
       <item>
        <layout class="QHBoxLayout" name="mainlayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>URL:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="urlEdit">
           <property name="text">
            <string>http://</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="threadlayout">
           <item>
            <widget class="QLabel" name="ThreadNumber">
             <property name="text">
              <string>Thread Number:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinThreadNumber">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Timeout">
             <property name="text">
              <string>Time Out(sec):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinTimeout">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="txtLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="code">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>HTTP Code:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbx200">
         <property name="text">
          <string>200 OK</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbx302">
         <property name="text">
          <string>302 Found</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbx403">
         <property name="text">
          <string>403 Forbidden</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="bottomLayout">
         <item>
          <widget class="QLabel" name="lblPercent">
           <property name="text">
            <string>0/0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="controllayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
